shared/actions/fs/index.js

```diff
@@ -320,6 +322,74 @@ const loadMimeTypeResult = (mimeType: string, action: FsGen.MimeTypeLoadPayload)
     })
   )
 
+const inboxQuery = {
+  computeActiveList: false,
+  readOnly: false,
+  status: Object.keys(RPCChatTypes.commonConversationStatus)
+    .filter(k => !['ignored', 'blocked', 'reported'].includes(k))
+    .map(k => RPCChatTypes.commonConversationStatus[k]),
+  tlfVisibility: RPCTypes.commonTLFVisibility.any,
+  topicType: RPCChatTypes.commonTopicType.chat,
+  unreadOnly: false,
+}
+
+function* loadResets(action: FsGen.LoadResetsPayload): Saga.SagaGenerator<any, any> {
+  // TODO: maybe uncomment?
+  // const conversations = yield Saga.call(
+  //   RpcChatTypes.localFindConversationsLocalRpcPromise,
+  //   action.payload.tlfName
+  // )
+  const resetRpc = new EngineRpc.EngineRpcCall(
+    {
+      'chat.1.chatUi.chatInboxUnverified': function*({
```

Comment:
do you really want only the unverified payload?

Reply:
I'm not sure what the differences are here. I was modeling off of an example in the chat index.js file.